programming/sr/servos/index.md

```diff
@@ -10,69 +10,72 @@ Servos
 This documentation refers to a feature which is only available on the physical robot kits.
 </div>
 
-The `servos` object is used to control a collection of Servo Boards.
-Similar to `motors` and `ruggeduinos`, `servos` can be used like a list.
-To do something with the **first Servo Board**, you would use:
+The `servo_board` object is used to control a collection of Servo Boards.
+
+When a single Servo Board is connected to your robot, you can control it
+using the `servo_board` object.
 
 ~~~~~ python
-R.servos[0].something...
+R.servo_board.something...
 ~~~~~
 
-...because indexes are 0-based (counting starts from 0, not 1).
-
-When you have more than one Servo Board connected to your kit
-they will be ordered based upon their serial number.
-
-The SR Part Code of each detected motor board is also printed to the log when your robot starts.
+The serial number of each detected Servo Board is printed to the log when your robot starts.
 It will look something like this:
 
 ~~~~~ not-code
-Found the following devices:
- - Servos:
-    0: Servo( serialnum = "SR0LG31" )
+sr.robot3.robot INFO - Found Student Robotics Servo Board v4 - srABC1
 ~~~~~
 
-
-However, like `motors` and `ruggeduinos`, `servos` is actually a dictionary.
-As a result, in `servos` you can also use the SR Part Code of the Servo Board as a key.
-For example, if you had a board that was labelled "SR0LG31",
-you could do this instead:
+If you have more than one Servo Board attached, you need to specify which one you want to control. This is done using the part number of the board. For example: if you had a board that was labelled "srABC1",
 
 ~~~~~ python
-R.servos["SR0LG31"].something...
+R.servo_boards["srABC1"].something...
 ~~~~~
 
+<div class="warning">
+    When you have more than one servo board connected to your kit,
+    you must use `R.servo_boards` and index by serial number. This is so
+    that the kit knows which servo board you want to control.
+</div>
+
```

Comment:
This is duplicate information.

Reply:
As much as it is duplicate, looking at it again, it would break the flow of the page if it was just the warning so I guess it's fine.